### PR TITLE
MAT-10138 Adding MeasureReviewStatus featureFlag

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-10138](https://jira.cms.gov/browse/MAT-10138)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: local-dev-env/serviceConfig.json [hunk 1]
```diff
@@ -36,6 +36,8 @@
     "testCaseSetId": false,
     "AdminUserList": false,
     "AdminUserProfile": false,
-    "usQualityCore": false
+    "usQualityCore": false,
+    "MeasureReviewStatus": true,
+    "LibraryReviewStatus": true
```
Comment (adongare) [inline local-dev-env/serviceConfig.json]: maybe we should default them to false incase we later forgot to set in external config. or doesn't matter?

Comment (RohitKandimalla) [reply local-dev-env/serviceConfig.json]: it doesn't matter, but I will set it to false anyways

**Review by adongare — APPROVED** [review]
